django/applications/catmaid/control/similarity.py

```diff
@@ -928,22 +928,22 @@ def compare_skeletons(request:HttpRequest, project_id) -> JsonResponse:
     config = NblastConfig.objects.get(project_id=project_id, pk=config_id)
 
     if not config.status == 'complete':
-        raise ValueError("NBLAST config #{} isn't marked as complete".format(config.id))
+        raise ValueError(f"NBLAST config #{config.id} isn't marked as complete")
 
     # Make sure we have a scoring matrix
     if not config.scoring:
-        raise ValueError("NBLAST config #{}  doesn't have a computed scoring.".format(config.id))
+        raise ValueError(f"NBLAST config #{config.id}  does not have a computed scoring.")
```

Comment:
Would you mind including the removal of the extra white-space in front of "does"?